js/battle.js

```diff
@@ -5331,6 +5331,7 @@ var Battle = (function () {
 		case 'c':
 			name = args[1];
 			if (this.ignoreSpects && (name.charAt(0) === ' ' || name.charAt(0) === '+')) break;
+			if (this.ignoreOpponent && name.charAt(0) === '★' && toUserid(name) !== app.user.get('userid')) break;
```

Comment:
You're not going to use the \unumbers form? I tend not to like Unicode in code files.